### PR TITLE
Fixes #8 : Make the album tiles + Singles list responsive for all screen sizes

### LINKED ISSUE #8: Make the site responsive
Make the `album tiles + Singles list` responsive for all screen sizes.

### PR DESCRIPTION
**Pull request by rishabhshuklax**
Fixes #8

### CODE REVIEW

**Comment by rishabhshuklax** [comment]
https://firela.github.io/Selena-Gomez-Discography/

File: css/offcanvas.css [hunk 1]
```diff
@@ -57,3 +57,26 @@ footer {
     width: 50%; /* 6 columns */
   }
 }
+@media screen and (max-width: 995px){
```
Comment (OrionStar25) [inline css/offcanvas.css:63]: You're on the right track! The album tiles are working for all screen sizes. But The singles' list is not visible for ipad or mobile devices. To check, you can use your browser developer tools or resize your screen.

File: index.html [hunk 2]
```diff
@@ -223,36 +223,36 @@ <h2><i>Kiss And Tell</i></h2>
...
         </div><!--/.col-xs-12.col-sm-9-->
 
-        <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar">
+        <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="float-none2 sidebar">
           <div class="list-group">
             <a href="#" class="list-group-item active">Singles ♭</a>
 
             <a href="https://www.youtube.com/watch?v=STO4-8vkG0U&list=RDSTO4-8vkG0U" class="list-group-item">Fetish</a>
-            <audio controls>
+            <audio controls class="audio">
```
Comment (OrionStar25) [inline index.html]: The `audio` class isn't working for smaller screen sizes. First, try to make the singles list visible for smaller screen sizes. Then we can make it responsive. Also, you might need to wrap the entire singles list into one `div` to make changes.

**Review by OrionStar25 — CHANGES_REQUESTED** [review]
:fire:

File: css/offcanvas.css (hunk 1 above)
Comment (rishabhshuklax) [reply css/offcanvas.css:63]: should i make singles list below the album tiles for small screen sizes?

**Comment by OrionStar25** [comment]
Yes that would be nice. Try to make it horizontal if you can. But that's
optional :)

File: css/offcanvas.css (hunk 1 above)
Comment (OrionStar25) [reply css/offcanvas.css:63]: Yes, that would be nice. You can also try to make it horizontal, but that's optional for now.

**Comment by OrionStar25** [comment]
@Firela You'll need to update your fork. Lot's of new changes. That'll help you remove merge conflicts too. :)

File: index.html [hunk 3]
```diff
@@ -197,71 +207,74 @@ <h2><i>A Year Without Rain</i></h2>
...
-   				 <a href="https://www.youtube.com/watch?v=sVEsO6reOGQ&index=5&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf">Crush</a>
-   				 <a href="https://www.youtube.com/watch?v=a_YR4dKArgo">Naturally</a>
-   				 <a href="https://www.youtube.com/watch?v=rZy_LBKaSrw&index=7&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf">The Way I Loved You</a>
-   				 <a href="https://www.youtube.com/watch?v=otHVZfN_s4k&index=8&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf">More</a>
-   				 <a href="https://www.youtube.com/watch?v=sM9H2k6qjNQ&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf&index=9">As A Blonde</a>
-   				 <a href="https://www.youtube.com/watch?v=OC7Zil4tBrA&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf&index=10">I Don't Miss You At All</a>
-   				 <a href="https://www.youtube.com/watch?v=4pMvpy99Ru4&index=11&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf">Stop And Erase</a>
-   				 <a href="https://www.youtube.com/watch?v=8XLatQRbdbo&index=12&list=PLopccIsGsvkCj7zqWjJ-bghxVnUm1Einf">I Got You</a>
-   				 <a href="https://www.youtube.com/watch?v=_RRyniZG0Jo">Tell Me Something I Don't Know</a>	
+   				 <a href="#">Kiss And Tell</a>
```
Comment (OrionStar25) [inline index.html]: You've gone back certain changes. That's not what we want. Try `git pull`

**Comment by pradeepgangwar** [comment]
@Firela As a good practice avoid using same id on more than one element in HTML. You can use classes for that purpose. We understand that this woks well but it's better to follow standards. Happy learning. 😄

File: index.html [hunk 4]
```diff
@@ -197,71 +207,74 @@ <h2><i>A Year Without Rain</i></h2>
...
  			<source src="songs/Kygo Selena Gomez - It Aint Me (with Selena Gomez) (Audio).mp3"
          	type='audio/mp3'>
   <!-- The next line will only be executed if the browser doesn't support the <audio> tag-->
  			<p>Your user agent does not support the HTML5 Audio element.</p>
 			</audio>
             
             <a href="https://www.youtube.com/watch?v=T2urfFpDX1c" class="list-group-item">Only You</a>
-             <audio controls>
- 			<source src="songs/only you.mp3"
+             <audio controls class="audio">
```
Comment (OrionStar25) [inline index.html]: Same as above.

File: css/offcanvas.css [hunk 5]
```diff
@@ -57,3 +60,30 @@ footer {
...
+    width: 100%;
+  }
+  #float-none1{
+    width: 70%;
+  }
+  #float-none2{
+    width: 30%;
+  }
+}
+@media screen and (max-width: 765px) {
```
Comment (OrionStar25) [inline css/offcanvas.css]: Getting closer. I can see the list but its overlapping.

**Review by OrionStar25 — CHANGES_REQUESTED** [review]
Keep going, almost there

**Comment by Divy123** [comment]
@Firela Please look into the link provided and try to follow along:
[https://help.github.com/articles/creating-a-pull-request/](url)

File: index.html [hunk 6]
```diff
@@ -176,7 +176,7 @@ <h2><i>When The Sun Goes Down</i></h2>
 				</div> 
                 </div><!--/.col-xs-6.col-lg-4-->
             
-            <div class="album col-xs-6 col-lg-4">
+            <div class="album col-xs-6 col-lg-4" id="float-none">
```
Comment (OrionStar25) [inline index.html:179]: change `float-none` to class.  Similarly for others.

File: index.html [hunk 7]
```diff
@@ -270,7 +271,7 @@ <h2><i>Kiss And Tell</i></h2>
 
 
 			<a href="https://www.youtube.com/watch?v=1Vn1BXfsd4Q" class="list-group-item">Trust Nobody</a>
-             <audio controls style="width: 100%;">
+             <audio controls  >
```
Comment (OrionStar25) [inline index.html]: Replace this with `<audio controls style="width: 100%;">`

**Review by OrionStar25 — CHANGES_REQUESTED** [review]
The album song lists overlap in mobile view.

File: index.html (hunk 6 above)
Comment (rishabhshuklax) [reply index.html:179]: changing it to class doesn't work because other classes overwrite the properties.

**Comment by rishabhshuklax** [comment]
> The album song lists overlap in mobile view.

should we make this another issue? because it's happening to albums above too!

File: index.html (hunk 6 above)
Comment (pradeepgangwar) [reply index.html:179]: Although this should not happen. I guess, you may be missing something. Anyways try using `!important` directive of CSS if other classes are overriding the property.

Comment (OrionStar25) [reply index.html:179]: Ill make this another issue.

**Review by OrionStar25 — APPROVED** [review]
Overall good job @Firela :smiley:

**Pull request merged**